### ISSUE TITLE
missing newline at the end -> crash

### ISSUE THREAD

**Issue by PavelVozenilek** [jfecher/ante#16]
I have file:

```
print "Start\n"

print "End\n"
```

where the last line **DOES NOT** end with newline.

The result:

```
O:\ante\ante-master\bin>ante test2.an
←[;3mtest2.an←[;m: ←[;1m3,7-15←[;0m     ←[;31merror: ←[;mCalled function was given 2 arguments but was declared to take
1
print "End\n"
←[;31m      ^^^^^^^^^←[;m

Compilation aborted.

```

**Comment by jfecher** [comment]
On it.  Also, you reminded me the color output doesn't work in windows cmd.exe, I just tested powershell and the coloring works there if you would like a more readable output.  I've also noticed you end your `print` calls with a '\n'.  `print` automatically prints a newline (it basically forwards a Str to `puts`), you can use `printne` if you want a print function that does not append a newline.

**Comment by PavelVozenilek** [comment]
This also happens with syntax errors:


```
O:\ante\ante-master\bin>ante test.an
←[;3mtest.an←[;m: ←[;1m4,8←[;0m ←[;31merror: ←[;msyntax error, unexpected ':', expecting '='
let x1 : u8 = 250
←[;31m       ^←[;m

Syntax error, aborting.

```

**Comment by jfecher** [comment]
`let x1 : u8 = 250` is invalid syntax in ante, that syntax error is justified.  As for the escape sequences, again those are just the (attempted) colored output.  You can use powershell for the time being while I find a colored output method that works with cmd.exe

**Comment by PavelVozenilek** [comment]
I found 
http://stackoverflow.com/questions/2048509/how-to-echo-with-different-colors-in-the-windows-command-line

> Windows before 10 - no native support for ANSI colors on the console
> 
> For Windows version below 10, the Windows command console doesn't support output coloring by default. You could install either Cmder, ConEmu, ANSICON or Mintty (used by default in GitBash and Cygwin) to add coloring support to your Windows command console.
> 
> Windows 10 - Command Line Colors
> 
> Starting from Windows 10 the Windows console support ANSI Escape Sequences and some colors by default.
> 

I use Win7.

**Comment by jfecher** [comment]
Ok, I fixed #17, but I will probably go about fixing this (#16) before I get into fixing the colors.  I expect there will be little difficulty in the latter.
